bin/license-checker-rseidelsohn

```diff
@@ -47,6 +47,7 @@ const usageMessage = [
     '   --start [filepath] path of the initial json to look for',
     '   --summary output a summary of the license usage',
     '   --unknown report guessed licenses as unknown licenses.',
+    '   --clarificationsFile A file that describe the license clarifications for each package, see clarificationExample.json, any field available to the customFormat option can be clarified. Can also be used to specify a subregion of a package\'s license file (instead reading the entire file).',
```

Comment:
As the help message states _All options in alphabetical order_, I'd prefer having this line in line 23 instead. Sorry for being picky.